lib/src/Base/Geom/DomainImplementation.cxx

```diff
@@ -89,6 +91,35 @@ Sample DomainImplementation::computeDistance(const Sample & sample) const
   return result;
 }
 
+GridLayout DomainImplementation::draw(const Sample& sample, const String& inColor, const String& outColor) const
```

Comment:
It was suggested to move it to `VisualTest.draw(domain, sample, inColor, outColor)`.